sdk/lib/core/date_time.dart

```diff
@@ -361,6 +361,16 @@ class DateTime implements Comparable<DateTime> {
     }
   }
 
+  /// Constructs a new [DateTime] instance
+  /// with Unix time epoch.
+  ///
+  /// The constructed [DateTime] represents 1970-01-01T00:00:00Z
+  /// ```dart
+  /// const unixEpochDay = DateTime.unixEpoch();
+  /// print(unixEpochDay); // 1970-01-01 00:00:00.000Z
+  /// ```
+  external const DateTime.unixEpoch();
```

Comment:
This should probably just be a constant (in UTC time).

Reply:
Thanks, Updated to be a constant!